nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/reporting/ReportLineageToAtlas.java

```diff
@@ -322,9 +328,19 @@
     private static final String ATLAS_PROPERTY_CLUSTER_NAME = "atlas.cluster.name";
     private static final String ATLAS_PROPERTY_REST_ADDRESS = "atlas.rest.address";
     private static final String ATLAS_PROPERTY_ENABLE_TLS = "atlas.enableTLS";
+    private static final String ATLAS_PROPERTY_TRUSTSTORE_FILE = "truststore.file";
```

Comment:
Could we use the Atlas constants instead?

Reply:
Replaced the literals with the Atlas constants but keeping the local constants because the Atlas names are not really straightforward.